spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/reactive/AbstractWebFluxEndpointHandlerMapping.java

```diff
@@ -298,7 +298,7 @@ protected interface LinksHandler {
 	@FunctionalInterface
 	protected interface ReactiveWebOperation {
 
-		Mono<ResponseEntity<Object>> handle(ServerWebExchange exchange, Map<String, String> body);
+		Mono<ResponseEntity<Object>> handle(ServerWebExchange exchange, Map<String, Object> body);
```

Comment:
Can you explain why you had to change this?

Reply:
I added a comment regarding these changes in the https://github.com/spring-projects/spring-boot/issues/42530#issuecomment-2457605930

The Map<String, String> request body does not support complex values like Map or List, only basic JSON types.

For example, this JSON is not supported at the moment:

```json
{
  "jobData": {
    "key": "value"
  }
}

```
due to
```
[org.springframework.http.converter.HttpMessageNotReadableException: 
JSON parse error: Cannot deserialize value of type `java.lang.String` from Object value (token `JsonToken.START_OBJECT`)]
```

Reply:
I meant that `@WebEndpointTest` with `JERSEY` infrastructure does support `Map<String, Object` as an endpoint parameter. 

If I revert my changes related to `@RequestBody Map<String, Object` the MVC and WebFlux will fail but Jersey will not. 

You can reproduce this issue by running this test `QuartzEndpointWebIntegrationTests.quartzTriggerJobWithCustomData`

Branch:  https://github.com/nosan/spring-boot/tree/42530-map-issue

Reply:
Thanks @nosan, I've created #43209 and reproduced the problem.